.coafile

```diff
@@ -66,3 +66,7 @@ cs_keywords =
 [PythonPackageInit]
 bears = PythonPackageInitBear
 ignore = *.py
+
+[yml]
+bears = YAMLLintBear
+files = *.yml, .ci/*.yml, tests/**/*.yml, bears/*.yml
```

Comment:
bears/**/*.yml will search out everything in bears dir but the yml files are only present in bears/*.yml
The next subdirs are only python dirs. So I guess it won't be needed, just may cause unnecessary checking.